setup.py

```diff
@@ -49,6 +49,10 @@
 # This can be a branch, tag, or commit
 PGVECTOR_COMMIT = 'v0.6.0'
 
+PGSPARSE_REPO = 'https://github.com/paradedb/paradedb.git'
+# This can be a branch, tag, or commit
+PGSPARSE_COMMIT = 'v0.5.4'
```

Comment:
Remember to add this to `_get_pg_source_stamp` to make CI build cache work properly.